### DISCUSSION TITLE
ignore `.js` extension in package.json's main

### DISCUSSION THREAD

**Discussion by Freed-Wu** [neoclide/coc.nvim#4919] [Ideas]
I notice some vscode extension's package.json's `main` can ignore `.js`, i.e., `"main": "out/extension"` means `"main": "out/extension.js"`. However, coc.nvim don't support it. Can coc.nvim support it?

**Comment by fannheyward** [comment]
Can't understand this, can you give an example?

**Reply by Freed-Wu** [reply]
```sh
yarn create coc-extension coc-test
cd coc-test
sed -i s=lib/index.js=lib/index= package.json
```

Then this extenstion cannot work, seem to like `lib/index` cannot be found.

In VS Code, it will search `lib/index.js` when `lib/index` is not found.

I just try to port some VS Code extension to vim and found this difference.
